sdk/communication/Azure.Communication.Calling.Server/tests/samples/Sample1_CallClient.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Communication.Identity;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.Communication.Calling.Server.Tests
+{
+    /// <summary>
+    /// Samples that are used in the README.md file.
+    /// </summary>
+    public partial class Sample1_CallClient : CallingServerLiveTestBase
+    {
+        public Sample1_CallClient(bool isAsync) : base(isAsync)
+        {
+        }
+
+        [Test]
+        [AsyncOnly]
+        [Ignore("Ignore for now as we get build errors that block checkingin.")]
```

Comment:
Can you clarify why there are build issues?

Reply:
https://teams.microsoft.com/l/message/19:f17d9cf475e74e1a94033cef0d612a67@thread.skype/1622562934962?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e9c1fc3-39df-4486-a26a-456d80e80f82&parentMessageId=1622562934962&teamName=Azure%20Communication%20Services&channelName=SDK&createdTime=1622562934962

We won not add live test in this PR and will fix it later.